materials/sql-aggregation.md

```diff
@@ -47,3 +47,43 @@ GROUP BY species_id, plot_id;
 * Aggregation functions remove null values from the calculations.
 
 > Do [Exercise 8 - COUNT]({{ site.baseurl }}/exercises/Aggregation-count-SQL).
+
+### `HAVING`
+
+* To filter by an outcome of an aggregation use `HAVING`
+
+```
+SELECT species_id, plot_id, MIN(weight), MAX(weight), AVG(weight)
+FROM surveys
+GROUP BY species_id, plot_id
+HAVING MIN(weight) IS NOT NULL;
+```
+
+* This works after aggregation, whereas `WHERE` works before aggregation
+
+```
+SELECT species_id, plot_id, MIN(weight), MAX(weight), AVG(weight)
+FROM surveys
+WHERE weight > 10
+GROUP BY species_id, plot_id
+HAVING MIN(weight) IS NOT NULL;
+```
+
+### Only grouped or aggregated fields in `SELECT`
+
+* Only grouped or aggregated fields should be used in `SELECT` when using `GROUP
```

Comment:
... 'fields that can be grouped or aggregated' ?. This statement is just a little unclear to me.